packages/animations/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## TBD
```

Comment:
Maybe just have the header say `## [1.0.1-dev] - TBD` and remove the explanation below. When we release, we can just remove the dev and insert the date and the next change after the release would just add the TBD header again.

(For the release version, we wouldn't want that explanation in the changelog.

Reply:
Also, then we can decide for every change we add which version number to bump (major.minor.bugfix).